util.py

```diff
@@ -173,17 +192,47 @@ def kill_robot(self, robot):
 
 class BeltTile(Tile):
     def __init__(self, direction, path, properties):
-        self.crossroads = properties[0]["value"]
-        self.belt_direction = properties[1]["value"]
-        self.move_count = properties[2]["value"]
+        self.direction_out = self.transform_direction(properties[0]["value"])
+        self.express = properties[1]["value"]
         super().__init__(direction, path, properties)
 
-    def move_robot(self, state):
-        # TO DO!
-
-        # 1) Express belts move 1 space
-        # 2) Express belts and normal belts move 1 space
-        pass
+    def transform_direction(self, direction_integer):
+        """
+        Method to transform the integer taken from json properties
+        to valid Direction or Rotation class instance for later processing.
+        """
+        if direction_integer == 0:
+            return Direction.N
+        if direction_integer == 90:
+            return Rotation.RIGHT
+        if direction_integer == -90:
+            return Rotation.LEFT
+        if direction_integer == 180:
+            return Rotation.U_TURN
+
+    def check_belts(self, express_belts):
```

Comment:
Jsem v této metodě ztracená (čti: nerozumím, co dělá): vysvětlíš, proč jsou podmínky uspořádané v tomto pořadí?

Reply:
Tato metoda slouží k výběru buď dvojitých pásů nebo všech pásů. Do metody vstupuje proměnná `express_belts`, která se nastaví při volání buď na `True`, pokud chci ověřit, že pás je dvojitý nebo `False` pokud mi nezáleží na tom, o jaký pás jde.
To jestli je pás dvojitý určuje argument políčka `express`. U políčka s dvojitým pásem je `True` u ostatních `False`.
První podmínka je pro dvojité pásy: argument express je `True` i hodnota vstupu je `True`. Metoda vrátí `True` tzn. našli jsme dvojitý pás.
Následný elif je pro případ, že hledám jakýkoliv pás. Na vstupu je tedy hodnota `False`. Metoda vrátí pro jakýkoliv pás `True`.
Poslední else zaručí, že metoda vrátí `False` v případě, že hledám dvojité pásy, ale narazím na obyčejný. 

Jinak tato metoda vrací `False` pro všechny ostatní políčka (viz nadtřída `Tile`).

Podmínky jsou uspořádané takto, aby se pokryli obě varianty vstupu a metoda se neukončila dřív než se zkontrolují obě varianty.